hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/api/Query.kt

```diff
@@ -1,17 +1,19 @@
 package uk.gov.justice.digital.hmpps.prisonersearch.search.services.attributesearch.api
 
-import com.fasterxml.jackson.annotation.JsonBackReference
 import io.swagger.v3.oas.annotations.media.Schema
+import org.opensearch.index.query.BoolQueryBuilder
+import org.opensearch.index.query.QueryBuilders
 import uk.gov.justice.digital.hmpps.prisonersearch.search.services.attributesearch.AttributeSearchException
+import uk.gov.justice.digital.hmpps.prisonersearch.search.services.attributesearch.api.JoinType.AND
+import uk.gov.justice.digital.hmpps.prisonersearch.search.services.attributesearch.api.JoinType.OR
 
 @Schema(description = "A query to search for prisoners by attributes")
 data class Query(
   @Schema(description = "The type of join to use when combining the matchers and subQueries", example = "AND", defaultValue = "AND")
-  val joinType: JoinType = JoinType.AND,
+  val joinType: JoinType = AND,
   @Schema(description = "Matchers that will be applied to this query")
   val matchers: List<TypeMatcher<*>>? = null,
   @Schema(description = "A list of sub-queries of type Query that will be combined with the matchers in this query")
-  @JsonBackReference
```

Comment:
Had to remove this to get serialization working due to recursion - which will break the OpenAPI docs. Will try and fix next.